<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="gene_pairkey_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GenePairKey</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>821</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verify_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="base64_en_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>base64_en</string>
    </property>
   </widget>
   <widget class="QPushButton" name="base64_de_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>base64_de</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
